.gitignore

```diff
@@ -1 +1,3 @@
 /Library/
+
+**/environment.js
```

Comment:
TODO: before deploying, we'll need some instructions for how the user can setup their own environment variables or some automated way to set their environment variables.